manifests/init.pp

```diff
@@ -3,38 +3,38 @@
 # awesomeness of packages.  The below class prepares the build environment.
 # For the configuration of the build environment, see Class[poudriere::env].
 class poudriere (
-  String $zpool                      = 'tank',
-  Stdlib::Absolutepath $zrootfs      = '/poudriere',
-  String $freebsd_host               = 'http://ftp6.us.freebsd.org/',
-  Stdlib::Absolutepath  $resolv_conf = '/etc/resolv.conf',
-  Boolean $ccache_enable             = false,
-  Stdlib::Absolutepath $ccache_dir   = '/var/cache/ccache',
-  $poudriere_base                    = '/usr/local/poudriere',
-  $poudriere_data                    = '${BASEFS}/data',
-  $use_portlint                      = 'no',
-  $mfssize                           = '',
-  $tmpfs                             = 'yes',
-  $distfiles_cache                   = '/usr/ports/distfiles',
-  $csup_host                         = '',
-  $svn_host                          = '',
-  $check_changed_options             = 'verbose',
-  $check_changed_deps                = 'yes',
-  String $pkg_repo_signing_key       = '',
-  $parallel_jobs                     = $facts['processors']['count'],
-  String $save_wrkdir                = '',
-  String $wrkdir_archive_format      = '',
-  String $nolinux                    = '',
-  String $no_package_building        = '',
-  String $no_restricted              = '',
-  String $allow_make_jobs            = '',
-  String $url_base                   = '',
-  String $max_execution_time         = '',
-  String $nohang_time                = '',
-  String $http_proxy                 = '',
-  String $ftp_proxy                  = '',
-  String $build_as_non_root          = '',
-  $environments                      = {},
-  $portstrees                        = {},
+  String                       $zpool                 = 'tank',
+  Stdlib::Absolutepath         $zrootfs               = '/poudriere',
+  String                       $freebsd_host          = 'http://ftp6.us.freebsd.org/',
```

Comment:
if we remove set this to `"https://downloads.freebsd.org"`
which is automatically selected based on geographical location

Reply:
Totally agree, I override this in my control-repo…  See my previous comment.